README.md

```diff
@@ -2,3 +2,10 @@
 A list of algorithms and datasets curated by the community. NHSX is not responsible for the contents of external links.
 
 If you are interested in adding a link to this list, [please raise an issue](https://github.com/nhsx/algorithm-library/issues/new) or submit a Pull Request.
+# Disclaimer:
+The content of this site is not owned or endorsed by NHSx. 
```

Comment:
The X is capitalised - or the Brand Police will be angered!  I suggest
"The content of links on this site are not owned or endorsed by NHSX.